lms/djangoapps/ccx/plugins.py

```diff
@@ -28,5 +29,7 @@ def is_enabled(cls, course, user=None):
             return True
```

Comment:
@macdiesel CCX app is inside a dynamic tab ``ccx coach`` inside courseware. When a user tries to access ``ccx coach`` directly with url we allow him to open ccx and after he opens app, first thing we check is that if this user is a valid coach on this ccx if not the send him error response:
 
``return HttpResponseForbidden(
                _('You must be a CCX Coach to access this view.'))``

cc @pdpinch